dusty/daemon.py

```diff
@@ -42,6 +42,12 @@ def _refresh_warnings():
         refresh_config_warnings()
         refresh_preflight_warnings()
 
+def _run_pre_command_functions(connection, suppress_warnings, client_version):
+    _refresh_warnings()
+    _send_warnings_to_client(connection, suppress_warnings)
+    check_and_load_ssh_auth()
```

Comment:
We should probably only do this if we've failed to get the socket previously. Also, refresh warnings should be the *last* thing we do in this block. Successfully acquiring the socket should clear the warning.